articles/architecture-scenarios/b2b/b2b-authentication.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Authentication
+description: Understand how authentication works in your CIAM implementation.
+toc: true
+topics:
+    - b2c
+    - ciam
+    - authentication
+    - universal-login
+contentType: concept
+useCase:
+    - authentication
+---
+# Authentication
+
+<%= include('../_includes/_authentication-intro.md') %>
+
+<%= include('../_includes/_bp-authentication.md') %>
+
+## Design considerations
+
+<%= include('../_includes/_authentication-design-considerations.md') %>
+* What do yuo do if you need to isolate my users by organization?
+* How do you handle identifying which organization your users' belong to?
+* What's the benefite of providing enterprise connections for your organization?
+
+<%= include('../_includes/_authentication-design-considerations2.md') %>
+
+Often companies need to segregate their users by organization and sometimes users can have access to more than one organization. Knowing which of these scenarios is relevant to your company will help define how to manage home realm discovery ([HRD](#home-realm-discovery)): whether you need to do it, when you need to do it, and how to accomplish it.
+
+## Universal Login
+
+<%= include('../_includes/_authentication-universal-login.md') %>
+
+## Home realm discovery
+
+Home realm discovery (HRD) is the process of identifying which identity provider the user belongs to *before* authenticating them. There are two ways HRD can occur: 
+
+* Provide a way for the user to tell the application
+* HRD through Universal Login
+
+Your system may need to do either or both methods so it is important to understand all approaches to HRD so that you can apply the one(s) that make the most sense to your applications. 
+
+:: panel Best practice
+If you don’t need to know ahead of time (for example, all of your users are in a shared user pool), then you don’t need to do HRD. You can allow users to authenticate first and then determine which organization they belong to using app metadata.
+
+### Application driven HRD
+
+A common and effective way for determining which realm a user belongs to is when an application is branded for each organization.  The organization has its own instance of the application.  This copy or instance can be physically isolated (running on a separate set of servers) or virtually (running on shared servers, but presented as if it could be isolated), and is generally denoted through either a custom hostname (companyA.application1.yourcompany.com) or path (application1.yourcompany.com/companyA).
+
+::: panel Best practice
+If your application already knows what connection (IDP) the user needs, then pass that along to the redirect to /authorize.
```

Comment:
If your application already knows what connection (IDP) the user needs, then pass that along to the redirect to /authorize using the connection query parameter.